src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,99 @@
+.cards {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
Move to * selector and remove from all other places.

Reply:
Not fixed. Move the `box-sizing` to the `*` selector to ensure a consistent box model throughout your page